hazelcast/src/main/java/com/hazelcast/map/impl/LocalMapStatsProvider.java

```diff
@@ -106,14 +110,14 @@ private void addOwnerPartitionStats(LocalMapStatsImpl localMapStats, String mapN
             lastUpdateTime = Math.max(lastUpdateTime, record.getLastUpdateTime());
         }
 
-        localMapStats.incrementOwnedEntryMemoryCost(ownedEntryMemoryCost);
-        localMapStats.incrementLockedEntryCount(lockedEntryCount);
-        localMapStats.incrementHits(hits);
-        localMapStats.incrementDirtyEntryCount(recordStore.getMapDataStore().notFinishedOperationsCount());
-        localMapStats.setLastAccessTime(lastAccessTime);
-        localMapStats.setLastUpdateTime(lastUpdateTime);
-        localMapStats.incrementHeapCost(recordStore.getHeapCost());
-        localMapStats.incrementOwnedEntryCount(recordStore.size());
+        localMapStatsImpl.incrementOwnedEntryMemoryCost(ownedEntryMemoryCost);
```

Comment:
What I often do to reduce the noise is to shorten names if it is clear from the surrounding context what is happening. E.g. consider renaming the variable to stats.